DNN Platform/Library/Services/Zip/readme.md

```diff
@@ -0,0 +1,11 @@
+﻿# Zip Services in DNN
+
+DNN uses SharpZipLib as the zip system of choice. Because of this, it is bundled with DNN. 
```

Comment:
I would prefer that this be included as comments in the `SharpZipLibRedirect` class, rather than as a markdown document.

Reply:
I prefer markdown - because often explanations will need multiple code files, but I'll put the same info into the class file as well - ok?

Reply:
For consistency with the rest of the project I would ONLY have this in the code comments.  Multi-line XML comments can support HTML formatting and will be automatically built into the documentation.  THis is used universally across the entire solution

Reply:
@mitchelsellers ok, I just moved all the docs into code-comments only